src/signer/canister/tests/it/canister/cycles_ledger/impls.rs

```diff
@@ -0,0 +1,19 @@
+use candid::Nat;
+
+use super::{Account, ApproveArgs};
+
+impl ApproveArgs {
+    /// Simple approval arguments.
+    pub fn new(spender: Account, amount: Nat) -> Self {
```

Comment:
Most of the time, ICRC2 approvals populate only these two values.  In our tests anyway.  Probably in real life as well.